<!doctype html>
<html lang="en">
    <head>
        <meta charset="utf-8">
        <meta http-equiv="x-ua-compatible" content="ie=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1">
        <title>gama's ramblings - Deep Variant summary</title>
        <link rel="stylesheet" href="../css/default.css" />
        <link rel="stylesheet" href="../css/syntax.css" />
        <link rel="stylesheet" href="../css/code.css" />
    </head>
    <body>
        <div id="header">
            <div id="logo">
                <a href="../">gama's ramblings</a>
            </div>
            <div id="navigation">
                <a href="../">Home</a>
                <a href="../about.html">About</a>
                <a href="../archive.html">Archive</a>
            </div>
        </div>

        <div id="content">
            <h1>Deep Variant summary</h1>
            <div class="info">
    Posted on July 18, 2017
    
        by Felix Raimundo
    
</div>

<h2 id="paper-report-creating-a-universal-snp-and-small-indel-variant-caller-with-deep-neural-networks">Paper report: Creating a universal SNP and small indel variant caller with deep neural networks</h2>
<p><a href="http://www.biorxiv.org/content/early/2016/12/14/092890">Creating a universal SNP and small indel variant caller with deep neural networks</a>[0]<br />
Ryan Poplin, Dan Newburger, Jojo Dijamco, Nam Nguyen, Dion Loy, Sam Gross, Cory Y. McLean, Mark A. DePristo<br />
DOI: https://doi.org/10.1101/092890</p>
<p>This paper, published in 2016, describes Googleâ€™s new Variant caller based on Deep learning methods.</p>
<p>What this new approach seems to bring are the following:</p>
<ul>
<li>Variant caller robust across sequencing technologies, whereas GATK requires special tuning for each sequencing technology or machine.</li>
<li>Better accuracy and F1 score than GATK.</li>
<li>Takes into account the fact that read errors are not independant.</li>
<li>Works accross species and chromosomes.</li>
</ul>
<p>Overall this method changes the game compared to GATK and seems to be able to fix many issues in the GATK way (described <a href="https://software.broadinstitute.org/gatk/best-practices/bp_3step.php?case=GermShortWGS">here</a>[1])</p>
<p>The way DeepVariant takes into account the errors is by a new representation of the reads. It represents them as an image where the aligned reads are represented as an image with each nucleic acid being color coded in a unique way.<br />
This allows to pass that image to a CNN and an <a href="https://arxiv.org/abs/1512.00567">inception-v2</a> architecture. The fact that it goes into a CNN is what allows it to take into account the neighbouring pixels and dealing with correlation in read errors. The paper mentions that a tensor-like representation approach is probably being currently investigated.</p>
<p>I imagine that the confidence in the quality of the reads is also taken into account in that representation even though not mentionned.</p>
<p>Even though not mentionned, it would be interesting to see how that approach would work if one was to work on unaligned data (even though that is a different problem), as this <a href="https://www.ncbi.nlm.nih.gov/pubmed/21903627">paper by H. Li describes</a>[3]</p>
<h3 id="references">References</h3>
<ul>
<li>[0] Creating a universal SNP and small indel variant caller with deep neural networks</li>
<li>[1] Best Practices for Germline SNP &amp; Indel Discovery in Whole Genome and Exome Sequence</li>
<li>[2] Rethinking the Inception Architecture for Computer Vision</li>
<li>[3] A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data.</li>
</ul>

        </div>

				<div id="disqus_thread"></div>
				<script>
				/**
				*  RECOMMENDED CONFIGURATION VARIABLES: EDIT AND UNCOMMENT THE SECTION BELOW TO INSERT DYNAMIC VALUES FROM YOUR PLATFORM OR CMS.
				*  LEARN WHY DEFINING THESE VARIABLES IS IMPORTANT: https://disqus.com/admin/universalcode/#configuration-variables*/
				/*
				var disqus_config = function () {
				this.page.url = PAGE_URL;  // Replace PAGE_URL with your page's canonical URL variable
				this.page.identifier = PAGE_IDENTIFIER; // Replace PAGE_IDENTIFIER with your page's unique identifier variable
				};
				*/
				(function() { // DON'T EDIT BELOW THIS LINE
				var d = document, s = d.createElement('script');
				s.src = 'https://gamazepsgithubio.disqus.com/embed.js';
				s.setAttribute('data-timestamp', +new Date());
				(d.head || d.body).appendChild(s);
				})();
				</script>
				<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>

        <div id="footer">
            Site proudly generated by
            <a href="http://jaspervdj.be/hakyll">Hakyll</a>
        </div>

    </body>
</html>
